### DISCUSSION TITLE
During incremental replication, the amount of storage required keeps on growing on the destination instance

### DISCUSSION THREAD

**Discussion by pratik36** [goharbor/harbor#18770] [Q&A]
Hello,
We have enabled replication between two Harbor instances that we have. It functions okay for the initial replication, but when it is performed every day, the size of the destination harbor progressively grows too large.

Source Harbor has total data of say 500 GB and on the destination Harbor it is now 2 TB+ after replication.

Need assistance to prevent the above space problem.

Both Harbor instances have the same version, v2.3.0.
Thank you!

**Comment by chlins** [comment]
Do you have any cleanup mechanism enabled like the tag retention or GC for the source harbor? These cleanup policies will not be replicated to the destination harbor and also you can setup the event-based replication policy and check the box `Delete remote resources when locally deleted` which will delete artifacts at destination if deletion operations happened on the source.

**Reply by pratik36** [reply]
Thank you @chlins for your response!
Just want to update you, I did check on this, and as there is no cleanup mechanism activated, the problem is not being caused by it.
Second, it appears that Harbor v2.3.0 does not have the option to "Delete remote resources when locally deleted."
